Packs/AutoFocus/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "AutoFocus",
     "description": "Use the Palo Alto Networks AutoFocus integration to distinguish the most\n  important threats from everyday commodity attacks.",
     "support": "xsoar",
-    "currentVersion": "1.3.10",
+    "currentVersion": "1.3.11",
```

Comment:
@yaakovi I think we should raise it to 1.4.0 or even 2.0. what do you think?

Reply:
2.0.0